docs/learningLab/introToLit/network.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 2
+---
+
+# 2. The Lit Network
+
+![image](../../../static/img/networkOverview.png)
```

Comment:
the img might have a typo. wondering if the copy should be:

"...Shares must **be** combined..."